jellyfin_kodi/downloader.py

```diff
@@ -282,6 +282,17 @@ def get_query_params(params, start, count):
             query['params'] = params
 
             result = result or {'Items': []}
+
+            # Mitigates #216 till the server validates the date provided is valid
+            for count in range(0, len(result["Items"])):
+                if result["Items"][count].get('ProductionYear'):
+                    try:
+                        from datetime import date
```

Comment:
Move the import to the top of the file.  Here it's being re-imported over and over again as each item is processed